server/service/service_fim_test.go

```diff
@@ -11,11 +11,16 @@ import (
 )
 
 func TestGetFIMService(t *testing.T) {
+	fileAccessesString := "[\"etc\", \"home\", \"cassandra\"]"
+	fileAccessStringValue := []string([]string{"etc", "home", "cassandra"})
```

Comment:
Nit: No need for outer `[]string()`, as you are already defining it as a `[]string` inside.